src/vaults/vaults.utils.ts

```diff
@@ -315,7 +315,8 @@ export async function loadVaultEventPerformances(
     throw new Error('Vault does not have adequate harvest history!');
   }
 
-  const measuredHarvests = recentHarvests.slice(0, 3);
+  const vault = await getCachedVault(vaultDefinition);
+  const measuredHarvests = recentHarvests.slice(0, recentHarvests.length - 1);
```

Comment:
maximizes harvests checking - this was reduced to see if it helped but quick harvests really still mess up so we should probably keep the full 3 weeks / continue to play with this values to see what is the best option (3 was too few)